trunk/NDHMS/Routing/Reservoirs/RFC_Forecasts/module_rfc_forecasts_tests.F

```diff
@@ -0,0 +1,96 @@
+module module_rfc_forecasts_tests
+    use module_rfc_forecasts
+
+contains
+
+    !Testing
+    function rfc_forecasts_data_info(rfc_forecasts_reservoir_data) result(rv)
+        implicit none
+        type (rfc_forecasts) :: rfc_forecasts_reservoir_data
+        logical :: rv
+        logical, dimension(1) :: ptr_state
```

Comment:
Should the dimension not be 4 here?

Reply:
You are correct. I am changing this to 4. I am also deleting the data_state and commented out data member checks, which I am adding in a future PR. @jdmattern-noaa fixing